server/controllers/expenses.js

```diff
@@ -225,15 +225,15 @@ export const pay = (req, res, next) => {
   }
 };
 
-function assertExpenseStatus(expense, status) {
-  if (status === status.PENDING && expense.status === status.APPROVED) {
+function assertExpenseStatus(expense, statusToConfirm) {
```

Comment:
Thought about that, `currentStatus` is technically what `expense.status` should be. We want to check that the current status of an expense matches a particular status. Open to better wording